docs/source/user_guide/supported_features.md

```diff
@@ -0,0 +1,20 @@
+# Supported Features
+
+| Feature                       | V0 | V1 |
+|-------------------------------|----|----|
+| Quantization                  |    |✅︎  |
+| LoRA                          |⚠️  |    |
+| Tensor Parallel               |✅︎  |✅︎  |
+| Prompt Adapter                |✅︎  |✅︎  |
+| Guided Decoding               |✅︎  |✅︎  |
+| Speculative Decoding          |    |✅︎  |
+| Pooling                       |✅︎  |    |
+| Multi Modality                |✅︎  |    |
+| LogProbs                      |✅︎  |✅︎  |
+| Prompt logProbs               |✅︎  |✅︎  |
+| Chunked Prefill               |    |    |
+| Automatic Prefix Caching      |✅︎  |✅︎  |
```

Comment:
I don't think we have prefix caching in either V0 or V1?

It's in the eventual plans for V1 though